bedrock/settings/base.py

```diff
@@ -2,13 +2,33 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# Django settings file for bedrock.
-
+import logging
+import os
 from os.path import abspath
+from django.utils.functional import lazy
 
-from funfactory.settings_base import *  # noqa
 from django.utils.http import urlquote
 
+from pathlib import Path
+
+from .static_media import *  # noqa
```

Comment:
My original plan was to have most or all of the static media related settings in there, but didn't end up doing that due to some requiring paths and I didn't want do to more path stuff in this file and circular imports and whatnot. But I left it this way to avoid confusion if people ever did want to add things. To me the `*` along with `# noqa` means that we know that this other file is really just an extension of this one. I'm fine either way really, but it's more consistent with how other settings imports work.